lib/api.js

```diff
@@ -122,6 +122,10 @@ function handleRestRequest (requestType, requestURI, requestData, callback) {
 				request_options.data = requestData;
 			}
 
+			if('del' === requestType && requestData){
```

Comment:
you can just combine this `if` with the one above.
`if (requestData && ('get' === requestType || 'del' === requestType)) {`